fuzzer/modules/sql_injection/strategies/lab6_string_interpolation.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import uuid
+
+VULNERABLE_ENDPOINT = 'http://127.0.0.1:4000/secret'
+
+
+def lab6_string_interpolation_test(host, grammar):
```

Comment:
This is the working test. Lots of things are statically defined here, but we should be able to re-use this pattern against our toy api.